app/index.html

```diff
@@ -340,12 +340,13 @@ <h2 class="title-margin-bottom">Media</h2>
                 <dt>Treasurer</dt>
                 <dd>Matthew Herndon '04</dd>
                 <dt>General Board Members</dt>
-                <dd>Valerie Derose Leimer '05</dd>
+                <dd><br>Valerie Derose Leimer '05</dd>

```

Comment:
I'm just going to remove it for now so we can deploy immediately. I think the UI looks better without this. If you feel strongly, let me know and we can add it back or discuss further.